server/kit/kitserver.go

```diff
@@ -101,39 +101,59 @@ func NewServer(svc Service) *Server {
 		lg.Log("error", err, "message", "exporter client encountered an error")
 	}
 	ocFlush := func() {}
-	if !observe.SkipObserve() && observe.IsGCPEnabled() {
-		exp, err := observe.NewStackdriverExporter(projectID, onErr)
-		if err != nil {
-			lg.Log("error", err,
-				"message", "unable to initiate error tracing exporter")
-		}
-		ocFlush = exp.Flush
-		trace.RegisterExporter(exp)
-		view.RegisterExporter(exp)
-
-		propr = &sdpropagation.HTTPFormat{}
+	if !observe.SkipObserve() {
```

Comment:
should `skipObserve` turn off all observation functionality? or should this be renamed to be something more SD-specific?

Reply:
Oh, yes. This disables all observability. I added a SD specific flag here https://github.com/nytimes/gizmo/pull/265/files#diff-87921a02aed8755cee4591d4f1c0511fR24

I'll add it here, too. Thanks

Reply:
Gotcha!

I get why the semantics are the way they are--preserving the default Stackdriver being enabled and Datadog disabled, but is any way we could simplify it to `[STACKDRIVER|DATADOG]_ENABLED` vs having `STACKDRIVER_DISABLED` and `DATADOG_ENABLED`?

Reply:
SD was enabled/initiated by default if the app was running in a suitable env, so without a breaking change, I think the only thing we can do is to add an option to disable it (until we migrate all the apps to DD).But even then, since it's an open source project, we may need to keep it as a default option.

Do you think we could do it in other way?

Reply:
Right, I think it makes sense to keep it enabled by default--I'm just commenting on the semantics of the environment variables that control it (having a `DISABLE` and an `ENABLE`) 

Maybe we could use the `envconfig` library in this package (since it's already in other parts of Gizmo) to configure it instead of just accessing the environment variables directly?

Reply:
Yes, you're right about the env vars and how they control the exporters.

Do you think looking for explicit values in those env variables would work?

Like `STACKDRIVER_ENABLED=false` (with default to `true`) to disable it and `DATADOG_ENABLED=true` (with default to `false`) to enable it? Plus reading all other values from the env automatically?

Reply:
Yes I think that would work here. I think we'd just need to have a config struct along the lines of:

```
type Conf struct {
    StackdriverEnabled   bool `default:"true" split_words:"true"`
    DatadogEnabled       bool `default:"false" split_words:"true"`
    ObservabilityEnabled bool `default:"true" split_words:"true"`
    ... (other config values, DD agent address, etc)
}
```

Reply:
That's great. Thanks.

I'll push an update.